modules/digital-twins/digital-twins-instance/endpoint--service-bus/main.bicep

```diff
@@ -92,3 +87,15 @@ output resourceGroupName string = resourceGroup().name
 
 @description('The name of the Endpoint.')
 output name string = endpoint.name
+
```

Comment:
As system assigned appears supported, let's add the corresponding principal ID output

Reply:
It does not work. Linter displays a warning and also gives me an error during deployment if I try this despite the warning...